packages/uniforms-bootstrap4/src/TextField.tsx

```diff
@@ -18,6 +18,7 @@ function Text(props: TextFieldProps) {
       className={classnames(props.inputClassName, 'form-control', {
         'is-invalid': props.error,
       })}
+      autoComplete={props.autoComplete}
```

Comment:
I've just checked and while both Bootstrap themes correctly understand `autoComplete` now, it also appears on the `div` rendered by `wrapField`. To fix that, do add `autoComplete` to the list of omitted values above.

Reply:
Fixed and added tests for that, hopefully it's okay now.